internal/pkg/template/templates/environment/cf.yml

```diff
@@ -432,6 +433,7 @@ Resources:
       LoadBalancerArn: !Ref InternalLoadBalancer
       Port: 443
       Protocol: HTTPS
+      SslPolicy: {{ if .PrivateHTTPConfig.SecurityPolicy }}{{ .PrivateHTTPConfig.SecurityPolicy }}{{ else }} ELBSecurityPolicy-2016-08 {{ end }} 
```

Comment:
Can we make the default behavior the same? aka if the field is not set we just don't set `SslPolicy` at all.

Reply:
+1